config/application.yml.example

```diff
@@ -27,8 +27,8 @@ queue_health_check_dead_interval_seconds: '240'
 # How often to enqueue simple jobs to make sure the background queues are running
 queue_health_check_frequency_seconds: '120'
 
-# Must be an even number. 16 seems to be what most sites use.
-recovery_code_length: '16'
+# The number of words in the recovery code phrase
+recovery_code_length: '12'
```

Comment:
that's a good question.

The math involves the size of the pool (number of words in the dictionary, regardless of their length in characters), whether elements in the phrase may repeat, and the length of the phrase.

The old way was a pool of 32 (26 alpha, 10 numeric), with repetition allowed, with a length of 16. That gives a possible number of combinations of `1503232609098`

https://www.hackmath.net/en/calculator/combinations-and-permutations?n=32&k=16&order=0&repeat=1

The new way has a pool of 1148 and a length of 12 for possible combinations of `10325563254528893828511284503`

https://www.hackmath.net/en/calculator/combinations-and-permutations?n=1148&k=12&order=0&repeat=0

To get the same brute force resistance with the new pool, we could shorten the length to 5 for `16471766935624` combinations.

(That calculator site is pretty fun to play with!)

I'm happy to lower it.